cli/popper/scm.py

```diff
@@ -1,30 +1,35 @@
 import os
 import popper.utils as pu
+import git
+import shutil
 
+try:
+    repo = git.Repo(search_parent_directories=True)
+except git.exc.InvalidGitRepositoryError:
+    pass
```

Comment:
Ignoring this exception might cause unexpected results. Since our assumption is that popper runs on git repos, I'd vote for failing if this is catched. In this way we also can avoid doing this `Repo` instantiation again

Reply:
Fixed it